### DISCUSSION TITLE
Error after Update: Declaration of Pimcore\Helper\CsvFormulaFormatter::unEscapeField(string $field)

### DISCUSSION THREAD

**Discussion by kopieMK** [pimcore/pimcore#16006] [Q&A / Support] [answered]
Hello people

After an upgrade from 10.6.6 to 10.6.9 I get the following error message after auditing the command `bin/console cache:clear`:

> In CsvFormulaFormatter.php line 22:
> Compile Error: Declaration of Pimcore\Helper\CsvFormulaFormatter::unEscapeField(string $field): string must be compatible with League\Csv\EscapeFormula::unescapeField(mixed $cell): mixed

The update did not give any error and went through successfully. The website itself is also running as far as I can tell.

I had also manually deleted the cache folders under var/cache/ once.
Command `bin/console pimcore:cache:clear` runs without problems.

Does anyone know this problem? Does jemand have a solution for it? It probably makes no sense to set the type definition itself to mixed right?

Kind regards
Magnus

**Comment by jdreesen** [comment, marked as answer]
This was fixed in Pimcore `11.0.10` (see #15999).

As Pimcore 10 is already EOL there will be no fix.

Apparently the issue occurs in `league/csv` `v9.11`, so you could prevent this version from being installed in your project (maybe by adding it as a composer `conflict`).

**Comment by kopieMK** [comment]
@jdreesen Thank you so much. I added the league/csv 9.11.0 as a conflict. Rerun the composer update and it works like a charm.

Have a nice day!

**Reply by braaam** [reply]
To help others with adding the conflict package to composer.json, these are the steps: 

Add following to your projects composer.json (note that newer versions still break at time of writing): 

`  "conflict": {
    "league/csv": ">=v9.11"
  },`

run "composer update"
